packages/utils/tool-utils/src/snapshotNormalizer.ts

```diff
@@ -70,11 +70,21 @@ function getDeepSortedObject(obj: any): any {
 }
 
 /**
- * Function that sorts a blob's content. If the content is an object or an array, deep sorts them.
- * @returns the sorted blob content.
+ * Function that normalizes a blob's content. If the content is an object or an array, deep sorts them.
+ * Special handling for certain runtime blobs, such as the "gc" blob.
+ * @returns the normalized blob content.
  */
-function getSortedBlobContent(content: string): string {
-    let sortedContent = content;
+function getNormalizedBlobContent(blobContent: string, blobName: string): string {
+    let content = blobContent;
+    if (blobName === gcBlobKey) {
+        // GC blobs may contain "unrefTimestamp" - The time the corresponding object became unreferenced. This is the
+        // timestamp of the last op processed and can differ between clients depending on when GC was run. It will be
+        // undefined if no ops were processed before running GC. So, remove it for the purposes of comparing snapshots.
+        const gcDetails = JSON.parse(content);
+        delete gcDetails.unrefTimestamp;
```

Comment:
If we don't want other clients to consume this data, can we avoid serializing it rather than counting on other clients to delete it?

Reply:
Do you mean we should not serialize this field at all? The reason it is removed from here is that this field won't be consistent from the point of view of comparing snapshots. The snapshot normalizer generates a version of the snapshot that is comparable with other snapshots.

Reply:
I see. Thanks.